operator-common/src/main/java/io/strimzi/operator/cluster/model/Ca.java

```diff
@@ -249,7 +248,7 @@ public static CertAndKey asCertAndKey(Secret secret, String key, String cert, St
         }
     }
 
-    public CertAndKey addKeyAndCertToKeyStore(String alias, byte[] key, byte[] cert) throws IOException {
+    public CertAndKey addKeyAndCertToKeyStore(Reconciliation reconciliation, String alias, byte[] key, byte[] cert) throws IOException {
```

Comment:
Should the Ca classes have `recocniliation` in constructor and as a instance field? Looks like there are many non-static methods using it.